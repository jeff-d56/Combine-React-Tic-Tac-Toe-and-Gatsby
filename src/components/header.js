import React from "react"

export const PureHeader = ({ data }) => (
  <header>
    <h1>{data.site.siteMetadata.title}</h1>
  </header>
)

export const Header = props => {
  const data = {
      site: {
        siteMetadata: {
          title: "Such wow. Very React.",
        },
      },
    }
  return <PureHeader {...props} data={data}></PureHeader>
}

export default Header