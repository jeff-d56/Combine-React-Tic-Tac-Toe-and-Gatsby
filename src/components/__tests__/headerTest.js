import React from "react"
import renderer from "react-test-renderer"

import { PureHeader as Header } from "../header"

describe("Header", () => {
  it("test if header renders correctly with snapshot", () => {
    const data = {
      site: {
        siteMetadata: {
          title: "Test if header works.",
        },
      },
    }
    const tree = renderer.create(<Header data={data} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})