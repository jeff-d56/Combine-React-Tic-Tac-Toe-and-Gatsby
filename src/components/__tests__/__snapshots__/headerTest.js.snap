// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header>
  <h1
    className="title"
  >
    Test if header works.
  </h1>
</header>
`;
